lib/couchdb.js

```diff
@@ -15,11 +15,15 @@ var url = require('url');
 var util = require('util');
 var _ = require('lodash');
 
+const DEFAULT_MODEL_VIEW = 'loopback__model__name';
+const DEFAULT_MODEL_PREFIX = 'LBModel';
+const DEFAULT_PROPERTY_PREFIX = 'LBIndex';
```

Comment:
And customizing it through `ENV` may not be enough, think it in this way...If a user want to customize those prefix, they may have different names per model, and per database.